webpack.config.js

```diff
@@ -34,14 +36,17 @@ module.exports = {
 		} ),
 		new AppCachePlugin(),
 		new HtmlWebpackPlugin( {
-			title: 'Simplenote',
-			templateContent: require( './index-builder.js' ),
-			inject: false
+			title: 'Simplenote'
 		} ),
 		new webpack.DefinePlugin( {
 			'process.env.NODE_ENV': JSON.stringify(
 				process.env.NODE_ENV || 'development'
-			)
+			),
+			config: JSON.stringify( config() )
```

Comment:
> I prefer configuration scripts as js instead of JSON usually.

I can accept this, though if you were to provide some rationale beyond "I like it" that would be even better :wink: